app/packages/looker/src/worker/async-labels-rendering-manager.ts

```diff
@@ -0,0 +1,199 @@
+import { Lookers } from "@fiftyone/state";
+import { v4 as uuid } from "uuid";
+import { ProcessSample } from ".";
+import { Coloring, Sample } from "..";
+import { LookerUtils } from "../lookers/shared";
+import { createWorker } from "../util";
+
+export type AsyncLabelsRenderingJob = {
+  sample: Sample;
+  labels: string[];
+  lookerRef: Lookers;
```

Comment:
I think this ref could be replaced a function that returns the required options. This would would exposing the internal `state`